website/docs/guides/release-notes.html.markdown

```diff
@@ -12,11 +12,40 @@ Track all Aviatrix Terraform provider's releases. New resources, features, and b
 
 ---
 
-``Last updated: R2.14.1 (UserConnect-5.4.1232)``
+``Last updated: R2.15 (UserConnect-6.0)``
 
 
 ---
 
+## 2.15.0
+### Notes:
+- Supported Controller version: **UserConnect-6.0**
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. New data sources:
+  - **data_source_aviatrix_firewall**
```

Comment:
Should these just be 
```
aviatrix_firewall
aviatrix_vpc_tracker
```

?

Reply:
I will update that and double check for existing Release Notes that use that incorrect syntax

Reply:
No. Data sources and resources are different.